EKF/optflow_fusion.cpp

```diff
@@ -63,8 +65,8 @@ void Ekf::fuseOptFlow()
 	// calculate the optical flow observation variance
 	const float R_LOS = calcOptFlowMeasVar();
 
-	float H_LOS[2][24] = {}; // Optical flow observation Jacobians
-	float Kfusion[24][2] = {}; // Optical flow Kalman gains
+	SparseVector24f<0,1,2,3,4,5,6> Hfusion; // Optical flow observation Jacobians
```

Comment:
Would move those declarations down to the place before it is used for the first time.